deltachat-ios/Extensions/Extensions.swift

```diff
@@ -84,3 +84,13 @@ extension UITableView {
         }
     }
 }
+
+extension UIFont {
+    static func preferredFont(for style: TextStyle, weight: Weight) -> UIFont {
```

Comment:
extension method that allows to add a weight to a dynamic system font, used in ChatListViewController to keep a medium font weight for the chat names